runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/graph/LengthPrefixUnknownCodersTest.java

```diff
@@ -144,4 +144,30 @@ public void test() throws IOException {
         CoderTranslation.fromProto(updatedCoderProto.getCoder(),
             RehydratedComponents.forComponents(updatedCoderProto.getComponents())));
   }
+
+  @Test
+  public void testIdempotent() throws IOException {
```

Comment:
It would be easier to just use the first test() and list out the explicit conversions that you feel the idempotent check is missing to the parameterized test data set.

Reply:
Done.